code/modules/clothing/suits/medium_armor.dm

```diff
@@ -402,6 +412,72 @@
 // CUSTOM //
 ////////////
 
+/obj/item/clothing/suit/armored/medium/combat/flatline/prototype
+	name = "prototype 'flatline' armor"
+	desc = "A very early model of the self-revival 'flatline' armor developed by none other than flatline himself. This one is highly customized. And by customized its cobbled together from junk, dripping algae and has a defib unit welded and installed on the chest piece. It looks like a scav made it."
+	icon_state = "flatline_prototype"
+	item_state = "flatline_prototype"
+	permeability_coefficient = 0.9
+	heat_protection = CHEST | GROIN | LEGS
+	cold_protection = CHEST | GROIN | LEGS
+	armor = list("melee" = 45, "bullet" = 35, "laser" = 35, "energy" = 35, "bomb" = 35, "bio" = 35, "rad" = 35, "fire" = 35, "acid" = 35, "wound" = 10)
```

Comment:
Why does the 'very early model' have more armor than the 'modified and refined' version?